control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1146,6 +1146,32 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/label/pools':
+    get:
+      tags:
+        - Pools
+      operationId: get_pools_topology
+      parameters:
+        - in: query
+          name: topology_label
+          required: true
+          schema:
+            $ref: '#/components/schemas/TopologyLabel'
```

Comment:
Should be a map?

Reply:
topology_label is a query param, and its passed as string  `A=B,C=D,E=F` . Cant pass map as a query param .